typed-racket-lib/typed-racket/private/parse-type.rkt

```diff
@@ -159,6 +159,7 @@
 (define-literal-syntax-class #:for-label Intersection)
 (define-literal-syntax-class #:for-label Refine)
 (define-literal-syntax-class #:for-label Self)
+(define-literal-syntax-class #:for-label Exist)
```

Comment:
Is `Exist` a general existential type?

If so, then I think it should accept parenthesis too, like All does `Exist (A) ....` (and someday, accept multiple names)
If not, then rename it to suggest why/how its different?

Reply:
Good point. My original intention was to add an existential function type but something like Exist-> didn't look right to me. On the other hand, Exist is not good either since it may imply TR enables standard pack/unpack. Any other thoughts? @bennn @samth

Reply:
`Exist (A ....) T` still sounds good. Can you draft docs for this type first, and pick a name after?